LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, Project Jupyter Contributors
+All rights reserved.
+
```

Comment:
I am not sure that we want to remove the previous copyright from the original file: 
```
- Copyright (c) 2001-2015, IPython Development Team
- Copyright (c) 2015-, Jupyter Development Team
```

Reply:
No problem, I will insert the two lines you indicated before line three of LICENSE.txt

Reply:
Added previous copyright lines from original COPYING.md file to LICENSE.txt.